Flazzy/Tags/DefineBitsJPEG3.cs

```diff
@@ -29,6 +29,11 @@ public DefineBitsJPEG3(HeaderRecord header, FlashReader input)
                 int partialLength = (2 + 4 + alphaDataOffset);
                 AlphaData = input.ReadBytes(Header.Length - partialLength);
             }
+            else if (Format == ImageFormat.PNG)
```

Comment:
Does every `ImageFormat` include alpha data even though it is empty, and is it always compressed with ZLIB? We could easily decompress them all here and get rid of these conditions.